docs/modules/ROOT/pages/includes/attributes.adoc

```diff
@@ -1,4 +1,4 @@
-:quarkus-version: 2.8.0.Final
+:quarkus-version: 2.10.1.Final
```

Comment:
does it really require 2.10?

Reply:
This is another change made during the build.